ui/src/components/Terminal/index.vue

```diff
@@ -75,8 +77,11 @@ const receivePostMessage = (): void => {
         }
         break;
       case WINDOW_MESSAGE_TYPE.PING:
-        lunaId.value = windowMessage.id;
         origin.value = e.origin;
+        lunaId.value = windowMessage.id;
+        protocol.value = windowMessage.protocol;
+
+        emits('update:protocol', protocol.value);
 
         sendEventToLuna(WINDOW_MESSAGE_TYPE.PONG, '', lunaId.value, origin.value);
         break;
```

Comment:
I'm sorry but it seems like there's no code to review here. Could you please provide some code or clarify what kind of code inspection you're referring too?